v22.2/export-data-with-changefeeds.md

```diff
@@ -5,7 +5,7 @@ toc: true
 docs_area: stream_data
 ---
 
-{% include_cached new-in.html version="v22.1" %} When you create an {{ site.data.products.enterprise }} changefeed, you can include the [`initial_scan = 'only'`](create-changefeed.html#initial-scan) option to specify that the changefeed should only complete a table scan. The changefeed emits messages for the table scan and then the job completes with a `succeeded` status. As a result, you can create a changefeed with `initial_scan = 'only'` to [export](export.html) data out of your database.  
```

Comment:
Oooh, I thought our GA release process was supposed to catch and delete these previous-version new-in's when we cut the next one. Surprised to see this here.

Reply:
Yeah... me too, not sure how that could have happened.